.github/workflows/bundle-diff.yml

```diff
@@ -0,0 +1,106 @@
+name: Bundle diff
```

Comment:
can we just put this all under the previous build one?

Reply:
Indeed we can. But as I mentioned in the linked issue, the pipeline does not work unfortunately, since there is no tool that can gather a rollup diff.